packages/react-components/src/molecules/ExportButton.tsx

```diff
@@ -34,31 +35,73 @@ const exportButtonStyles = css({
     },
 });
 
+const infoWrapperStyle = css({
+  verticalAlign: 'middle',
+});
+
+const tooltipStyle = css({
+  textAlign: 'left',
+});
+
 const exportIconStyles = css({ display: 'flex' });
 type ExportButtonProps = {
   readonly exportResults?: () => Promise<void>;
+  readonly buttons?: Array<{
+    buttonText: string;
+    errorMessage: string;
+    exportResults?: () => Promise<void>;
+  }>;
+  readonly info?: ReactNode;
 };
 
-const ExportButton: React.FC<ExportButtonProps> = ({ exportResults }) => {
+const ExportButton: React.FC<ExportButtonProps> = ({
+  exportResults,
+  info,
+  buttons,
+}) => {
+  const defaultButton = {
+    exportResults,
+    buttonText: 'CSV',
+    errorMessage: 'There was an issue exporting to CSV. Please try again.',
+  };
+  const buttonGroup = (buttons?.length ? buttons : [defaultButton]).filter(
+    (button) => !!button.exportResults,
+  );
+
   const toast = useContext(ToastContext);
-  return exportResults ? (
+
+  return buttonGroup.length ? (
     <span css={exportSectionStyles}>
-      <strong>Export as:</strong>
-      <Button
-        noMargin
-        small
-        onClick={() =>
-          exportResults().catch(() => {
-            toast('There was an issue exporting to CSV. Please try again.');
-          })
-        }
-        overrideStyles={exportButtonStyles}
-      >
-        <>
-          <div css={exportIconStyles}>{ExportIcon}</div>
-          CSV
-        </>
-      </Button>
+      <strong>
+        Download:
+        {info ? (
+          <TooltipInfo
+            overrideWrapperStyles={infoWrapperStyle}
+            overrideTooltipStyles={tooltipStyle}
+          >
+            {info}
+          </TooltipInfo>
```

Comment:
very minor, but I wonder if we couldn't use the `Tooltip` instead of `TooltipInfo` and have the tooltip appearing on hover instead of when we click on the tooltip, like in here

https://github.com/yldio/asap-hub/blob/master/packages/react-components/src/organisms/ManuscriptCard.tsx#L429:L440